docs/dictionary/command/read-from-process.lcdoc

```diff
@@ -98,7 +98,7 @@ write to process (command), read from driver (command), EOF (constant),
 function (control structure), openProcesses (function), ticks (function),
 result (function), milliseconds (function), platform (glossary),
 binary file (glossary), variable (glossary), byte (glossary), 
-case-sensitive (glossary), expression (glossary), it variable (glossary),
+case-sensitive (glossary), expression (glossary),
```

Comment:
It variable has been correctly removed here but the reference to it needs to be removed in line 64 to read:

> The data is placed in the \<it\> \<variable\> after reading. If you specified

This then refers to the 'it' keyword and 'variable' glossary entries.